### ISSUE TITLE
Docker images outdated?

### ISSUE THREAD

**Issue by bhupiister** [advancedtelematic/ota-community-edition#127]
Docker images are 9 years old. I am sure it must be missing some of the features.
Any plans to update it? 

Is this project maintained regularly?

**Comment by houcros** [comment]
> Is this project maintained regularly?

Nope, see [this conversation](https://github.com/advancedtelematic/ota-community-edition/issues/105#issuecomment-868379000).
